staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -155,39 +155,39 @@ type KubeSchedulerProfile struct {
 // be invoked before default plugins, default plugins must be disabled and re-enabled here in desired order.
 type Plugins struct {
 	// QueueSort is a list of plugins that should be invoked when sorting pods in the scheduling queue.
-	QueueSort *PluginSet `json:"queueSort,omitempty"`
```

Comment:
revert the changes in this file. It is actually convenient to have pointers here so that we can really "omitempty".
We might need custom converstion. We will see. Revert this file first.